loader.py

```diff
@@ -1,3 +1,182 @@
+import logging
+from os import walk
+from os.path import isdir, join, splitext
+from util import *
+
+logger = logging.getLogger(__name__)
+
+"""
+interface Organizer:
```

Comment:
da sa pouzit `abc` meta classu a tak definovat akysi interface. 
```python
class Organizer(metaclass=abc.ABCMeta):
    @abc.abstractmethod
    def my_interface_method():
         # default behaviour or pass

class MyCoolOrganizer(Organizer):
    ...
```

Reply:
Viem o tom. Potreboval som sa rychlo rozpisat a pre takyto maly projekt to nebolo treba. Ale je to dobry practice.